zerver/openapi/zulip.yaml

```diff
@@ -4740,6 +4740,16 @@ paths:
             type: boolean
             default: false
           example: true
+        - name: is_web_public
+          in: query
+          description: |
+            This parameter determines whether any newly created streams will be
+            web public streams.
+            TODO: Add a doc link here to explain what are web-public-streams.
```

Comment:
We should write documentation explaining web public stream, web public visitors and web public view.

Reply:
Since the specific details for these regarding permission are not super clear yet, I withheld from creating these.